### PR TITLE
Jordy add notificationController unit tests

### PR DESCRIPTION
**Pull request by j-corp-25**
# Description

Unit tests for notificationController

## Related PRS (if any):
N/A

## Main changes explained:

- Unit tests for the getUserNotifications endpoint
- Unit tests for the createUserNotifications endpoint
- Unit tests for the deleteUserNotifications endpoint

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. run `npm run test notification`

**If  you want a detailed description you can run `npm run test notification -- --verbose`**

## Screenshots or videos of changes:

userNotificationControllerUnitTests

deleteUserNotification

createUsernotification

## Note:

When I was working on the unauthorized method, I corrected the spelling on the controller.

### CODE REVIEW

**Review by linkhanthtel — APPROVED** [review]
All tests passed as expected! Nice work!

**Review by Alakabtaha0 — APPROVED** [review]
Passed all 3 tests. good job.
Screenshot 2024-04-23 at 18 33 59

File: requirements/notificationController/getUserNotification.md
Comment (mianmiantea2019) [inline requirements/notificationController/getUserNotification.md:1]: It's recommended to test first negative cases first. So please reorganize your .md file to include the negative cases first and positive cases at last.

**Review by mianmiantea2019 — CHANGES_REQUESTED** [review]
Hi Jordy, Good work!  Just few recommendations in the .md file

image

File: requirements/notificationController/getUserNotification.md
Comment (j-corp-25) [reply requirements/notificationController/getUserNotification.md:1]: Thank you for pointing this out! I have updated the order of the test cases : )

**Review by j-corp-25 — COMMENTED** [review]
updated order of tests base on Christy's suggestion

**Review by mianmiantea2019 — APPROVED** [review]
Good work!

**Review by Abi-Liu — APPROVED** [review]
The tests are looking good and are all passing. Good work!

Screenshot 2024-04-26 at 3 14 51 PM

**Review by gowthamdongari — APPROVED** [review]
test cases sucessfully 
image

**Pull request closed without merging**